extension/main.js

```diff
@@ -1,6 +1,11 @@
 (function () {
 	'use strict';
 
+	function createNotificationId() {
+		var id = Math.floor(Math.random() * 9007199254740992) + 1;
```

Comment:
It's not clear from the docs if notification ID should be unique. Therefore I just took this snippet from the example in docs.